charts/ceph-csi-cephfs/templates/provisioner-service.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if semverCompare "<=1.13" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
even if the HTTP metrics and grpc metrics are not enabled we are creating a service for it here

Reply:
See #81

Reply:
AFAIK we don't need the service to be created, in the stateful set you need to mention a dummy service name.

can you check it